.github/workflows/codeql-analysis.yml

```diff
@@ -50,21 +50,30 @@ jobs:
         # Prefix the list here with "+" to use these queries and those in the config file.
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
+    - uses: actions/cache@v4
+      with:
+        path: |
+          ~/.cache/go-build
+          ~/.cache/pip
+          ~/go/pkg/mod
```

Comment:
@piyush-garg why do we cache `pip` things?
Does `CodeQL Analysis` uses python?

Reply:
cood catch, i think we dont need pip cache, removing this